content/blog/accessing-dtap-in-pods.md

```diff
@@ -0,0 +1,264 @@
+
+1. Add ```/opt/bdfs/bluedata-dtap.jar``` to the classpath.
+2. Configure Hadoop with the following values.
+
+| name | value |
+|---|---|
+| fs.dtap.impl | com.bluedata.hadoop.bdfs.Bdfs |
+| fs.AbstractFileSystem.dtap.impl | com.bluedata.hadoop.bdfs.BdAbstractFS |
+| fs.dtap.impl.disable.cache* | false |
+
+*optional
+
+> Reference:
+> - [Accessing DataTaps in Kubernetes Pods](https://docs.containerplatform.hpe.com/53/reference/kubernetes/tenant-project-administration/datataps/Accessing_DataTaps_in_Kubernetes_Pods.html)
+
+
+## Uniform Resource Identifier
+
+In HPE Ezmeral Container Platform, you can see different types of file system used by the shared storage resources. You can manage different data source through a GUI and representing files with same URI. The URI will be in the format of 
+
+```
+dtap://datatap_name/some_subdirectory/another_subdirectory/some_file
+```
+
+|Screenshot|Description|
+|---|---|
+|![image](https://user-images.githubusercontent.com/72959956/121467168-35150680-c9eb-11eb-901c-77e83097cdf9.png)| You can manage different data source whether they are in MapR filesystem or HDFS. |
+|![image](https://user-images.githubusercontent.com/72959956/121467262-5f66c400-c9eb-11eb-958d-911f18281a27.png)| You can upload, delete or rename files using GUI. |
+
+
+# Access DataTaps using HDFS commands
+
+## Introduction
+
+The Hadoop distributed file system (HDFS) is the key component of the Hadoop ecosystem. HDFS commands, of course, are the commands which responsible for manipulate files for HDFS. 
+
+To use the HDFS commands, first you need to start the Hadoop services using the following command:
+
+
+
+## Prepare Hadoop
+
+Some of the KubeDirector App provided by HPE is pre-installed a well-configured Hadoop for you. Hence, the following installation steps can be skipped.
+
+
+### Install OpenJDK and the dependency
+
+
+```bash
+apt update && apt upgrade -y
+apt install wget -y
+
+# install openjdk
+DEBIAN_FRONTEND=noninteractive apt-get install openjdk-11-jdk-headless -y
+```
+
+### Download Hadoop and untar Hadoop
+
+You can always find the latest version of Hadoop on [Apache Hadoop Releases](https://hadoop.apache.org/releases.html).
+
+
+```bash
+wget https://apache.website-solution.net/hadoop/common/hadoop-3.3.0/hadoop-3.3.0.tar.gz   # Download Hadoop binary
+tar zxf hadoop-*.tar.gz                                                                   # Untar Hadoop binary
+mv hadoop-3.3.0 $HOME/hadoop                                                              # Rename and move Hadoop folder to $HOME
+cd $HOME/hadoop                                                                           # Move directory to hadoop
+```
+
+
+
+### Configure the required environment
+
+In ```$HADOOP_HOME/etc/hadoop/hadoop-env.sh``` file, assign the following environment variables (```$JAVA_HOME```, ```$HADOOP_HOME```, ```$HADOOP_CLASSPATH```):
+
+```bash
+# These two variables is needed for HDFS command. Located at line 54, 58.
```

Comment:
Is this line actually supposed to be in the black code block? It seems like it might actually be text that is supposed to be just ahead of the code block. Just double checking. If it is correct as it is shown in the preview, just mark my comment here as resolved.

Reply:
those are the file name and the variable, just wanna emphasis to the readers, maybe  I can change it to italic